services/apigatewayv2/helm/templates/role-writer.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: ack-apigatewayv2-writer
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups:
+  - apigatewayv2.services.k8s.aws
+  resources:
+  - apis
```

Comment:
nit: extra lines here and in a few other templates, is this expected?

Reply:
nice catch. I can fix this up.

Reply:
I will fix this in a future PR.